specifications/history-to-releases.org

```diff
@@ -0,0 +1,112 @@
+5473    2020-09-11T12:55:28Z  s390x    4.3                       edge, candidate*, stable*
+5472    2020-09-11T12:50:35Z  amd64    4.3                       edge, candidate*, stable*
+
+#+END_SRC
+
+And this is how the new output shall display:
+
+#+BEGIN_SRC
+Rev.    Uploaded              Arches    Version                   Channels
+5560    2020-10-01T23:52:20Z  armhf     4.3.post16+gitaaf24afc    latest/edge*
+5559    2020-10-01T23:42:21Z  arm64     4.3.post16+gitaaf24afc    latest/edge*
+5558    2020-10-01T23:21:20Z  ppc64el   4.3.post16+gitaaf24afc    latest/edge*
+5557    2020-10-01T23:19:26Z  i386      4.3.post16+gitaaf24afc    latest/edge*
+5556    2020-10-01T23:14:23Z  s390x     4.3.post16+gitaaf24afc    latest/edge*
+5555    2020-10-01T23:09:20Z  amd64     4.3.post16+gitaaf24afc    latest/edge*
+5554    2020-10-01T13:51:02Z  amd64     4.3.post24+git2b31c74     latest/edge/pr-3299*
+5553    2020-10-01T12:33:26Z  amd64     4.3.post18+git05d646c     latest/edge/pr-3303*
+5552    2020-10-01T00:25:44Z  amd64     4.3.post18+gitb7dba6f     latest/edge/pr-3302*
+5551    2020-09-30T23:37:19Z  armhf     4.3.post15+gitdde6b5f4    latest/edge
+5550    2020-09-30T23:14:01Z  arm64     4.3.post15+gitdde6b5f4    latest/edge
+5549    2020-09-30T22:59:20Z  i386      4.3.post15+gitdde6b5f4    latest/edge
+5548    2020-09-30T22:55:29Z  ppc64el   4.3.post15+gitdde6b5f4    latest/edge
+5547    2020-09-30T22:44:22Z  s390x     4.3.post15+gitdde6b5f4    latest/edge
+5546    2020-09-30T22:39:37Z  amd64     4.3.post15+gitdde6b5f4    latest/edge
+5545    2020-09-30T22:12:11Z  amd64     4.3.post23+git5492f37     latest/edge/pr-3299
+5544    2020-09-29T23:46:10Z  amd64     4.3.post16+git20374d3     latest/edge/pr-3302
+5543    2020-09-29T19:57:46Z  amd64     4.3.post15+git3d475a7     latest/edge/pr-3301*
+5542    2020-09-29T12:44:04Z  amd64     4.3.post19+git2612da8     latest/edge/pr-3241*
+5541    2020-09-29T12:43:03Z  amd64     4.3.post19+git2612da8     latest/edge/pr-3241
+...
+5477    2020-09-11T18:41:18Z  armhf     4.3                       latest/stable*,latest/candidate*,latest/edge
+5476    2020-09-11T15:50:57Z  arm64     4.3                       latest/stable*,latest/candidate*,latest/edge
+5475    2020-09-11T13:11:08Z  i386      4.3                       latest/stable*,latest/candidate*,latest/edge
+5474    2020-09-11T13:05:33Z  ppc64el   4.3                       latest/stable*,latest/candidate*,latest/edge
+5473    2020-09-11T12:55:28Z  s390x     4.3                       latest/stable*,latest/candidate*,latest/edge
+5472    2020-09-11T12:50:35Z  amd64     4.3                       latest/stable*,latest/candidate*,latest/edge
+#+END_SRC
+
+* Implementation
+
+The Snap Store API for [[https://dashboard.snapcraft.io/docs/v2/en/snaps.html#snap-releases][Releases]] returns two /streams/ of interest, those
+are =revisions= and =releases=. These streams are ordered and relied on
```

Comment:
ordered how?  I don't think you need to state that it's relied on (anything in the API must be reliable).